reference/conanfile/tools/cmake/cmaketoolchain.rst

```diff
@@ -200,7 +200,7 @@ There are two ways of providing a custom CMake toolchain file:
             # tool_requires
             user_toolchains = []
             for dep in self.dependencies.direct_build.values():
-                ut = dep.conf_info["tools.cmake.cmaketoolchain:user_toolchain"]
+                ut = dep.conf_info.get("tools.cmake.cmaketoolchain:user_toolchain")
```

Comment:
Actually, this use case in the docs is probably irrelevant now? No need that recipes do weird things with this to aggregate a list, this is exactly the reason this feature was introduced?

Reply:
I don't know what you mean. I only changed the way to get the value (it does not matter if it's a list or not), I was only updating it.

Reply:
Yes, sure, not part of your code change. But it would make sense to change or to remove that specific example from the docs, because that doesn't make sense anymore. It was a workaround for the lack of lists in conf.